audit/src/main/java/org/wildfly/security/audit/SyslogAuditEndpoint.java

```diff
@@ -116,6 +142,8 @@ public static Builder builder() {
         private boolean tcp = true;
         private String hostName;
         private SocketFactory socketFactory = null;
+        private SyslogHandler.SyslogType format = SyslogHandler.SyslogType.RFC5424;
+        private int maxReconnectAttempts = -1;
```

Comment:
Looks like ```maxReconnectAttempts``` defaults to -1 here but defaults to 0 in Core. Is there a reason for this?

Reply:
Forgot to update this when the default value was changed, fixed now